Sage/carbon
# feat(loader, loader-bar): describe loader and loader-bar components using design tokens - FE-4452

### Proposed behaviour

Describes the loader and loader-bar components using design tokens.
Updates tests after making the above changes.

### Current behaviour

The loader and loader-bar components use the theme styled-component property.

### Checklist




#### QA


### Additional context



### Testing instructions

The correct appearance can be checked in the component: Loader, Loader Bar.
For theme: mint and aegean there should be no differences.
Use devtools to check if loaders use css variables.
